src/MetaModels/Attribute/Timestamp/Timestamp.php

```diff
@@ -54,7 +58,7 @@ public function getFieldDefinition($arrOverrides = array())
         $strDateType                       = $this->getDateTimeType();
         $arrFieldDef                       = parent::getFieldDefinition($arrOverrides);
         $arrFieldDef['eval']['rgxp']       = $strDateType;
-        $arrFieldDef['eval']['datepicker'] = ($strDateType !== 'time');
+        $arrFieldDef['eval']['datepicker'] = !empty($arrFieldDef['eval']['readonly'])?false:true;
```

Comment:
Please change this to `$arrFieldDef['eval']['datepicker'] = empty($arrFieldDef['eval']['readonly']);` as the result is already boolean.

Since when is time picker supported? Do we need to bump the minimum version in `composer.json` along the way?